vcpkg/commands/format-manifest.md

```diff
@@ -0,0 +1,54 @@
+---
+title: vcpkg format-manifest
+description: Reference for the vcpkg format-manifest command. Formats vcpkg.json files and converts CONTROL files to vcpkg.json format.
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 08/17/2023
+ms.prod: vcpkg
+---
+
+# vcpkg format-manifest
+
+## Synopsis
+
+```console
+vcpkg format-manifest [--all] [--convert-control] [<path>...]
+```
+
+## Description
+Formats manifest (`vcpkg.json`) files and converts control (`CONTROL`) files into the manifest format.
+
+>[!NOTE]
+>If both `CONTROL` and `vcpkg.json` file are present in the same directory, the command will exit with an error.
+
+## Examples
+
+To format all manifest files in the ports directory:
```

Comment:
Which ports directory?

Reply:
```suggestion
To format all manifest files in the built-in `ports` directory:
```

Some of these terms will have supporting "concepts" documentation so that we can call out vcpkg-specific terms without having to explain them every single time.

Reply:
Agreed, but even as an "advanced" vcpkg user, I could easily be confused by this. Is this the ports directory from the vcpkg instance, the ports directory in my custom registry, or a "ports/" directory in my project repo which I use for overlays?